src/sources/mongodb_metrics/mod.rs

```diff
@@ -1024,12 +1024,12 @@ mod tests {
 
     #[test]
     fn generate_config() {
-        crate::test_util::test_generate_config::<MongoDBMetricsConfig>();
+        crate::test_util::test_generate_config::<MongoDbMetricsConfig>();
     }
 
     #[tokio::test]
     async fn sanitize_endpoint_test() {
-        let endpoint = "mongodb://myDBReader:D1fficultP%40ssw0rd@mongos0.example.com:27017,mongos1.example.com:27017,mongos2.example.com:27017/?authSource=admin&tls=true";
+        let endpoint = "mongodb://myDbReader:D1fficultP%40ssw0rd@mongos0.example.com:27017,mongos1.example.com:27017,mongos2.example.com:27017/?authSource=admin&tls=true";
```

Comment:
Will this affect execution of the test?

Reply:
I actually meant to change this back, but I see I didn't 👍